src/signals/incident/components/form/Header/index.js

```diff
@@ -47,14 +38,14 @@ const Header = ({ className, meta, options, touched, hasError, getError, childre
   return (
     <Wrapper className={className} invalid={containsErrors}>
       {meta?.label && (
-        <Label>
+        <Label htmlFor={meta.id}>
           {meta.label}
 
           {isOptional && <Optional>(optioneel)</Optional>}
         </Label>
       )}
 
-      {meta?.subtitle && <SubTitle>{meta.subtitle}</SubTitle>}
+      {meta?.subtitle && <SubTitle id={`subtitle-${meta.id}`}>{meta.subtitle}</SubTitle>}
```

Comment:
Better to double check that the id is passed to every component (also see comment below). Right now this results in `id="subtitle-undefined"` in the location map.